app/src/test/java/com/nervousfish/nervousfish/ExampleUnitTest.java

```diff
@@ -13,17 +21,27 @@
  */
 @SuppressWarnings("PMD")
 public class ExampleUnitTest {
+    ISample sample = mock(ISample.class);
+
     @Test
     public void addition_isCorrect() throws Exception {
         assertEquals(4, 2 + 2);
     }
 
+    @Test
+    public void bar() {
+        when(sample.foo()).thenReturn(5);
+        assertEquals(new SampleClass(sample).get(), 5);
+        Constants constants = (Constants) accessConstructor(Constants.class, mock(IServiceLocatorCreator.class));
```

Comment:
It is an example of `accessConstructor`, if no error is thrown it should be working